src/superface-test.interfaces.ts

```diff
@@ -54,6 +55,8 @@ export type ProcessingFunction = (
 ) => Promise<void> | void;
 
 export interface RecordingProcessOptions {
+  prepare?: boolean;
```

Comment:
nitpick: this could be typed `kind?: 'prepare' | 'teardown'` right? It is not like you can have both true 🤔

Reply:
Kind is better for extension.

![image](https://user-images.githubusercontent.com/959390/198390414-7a6409dc-ff62-4471-90d0-4215db0d9bd2.jpeg)